python/tchannel/tornado/stream.py

```diff
@@ -98,27 +103,34 @@ def __init__(self, buf=None, auto_close=True):
 
         :param buf: the buffer for the in memory stream
         """
-        self._stream = deque()
+        self.stream = deque()
```

Comment:
No. Consumers don't get access to the `deque`. That's an implementation detail. We've already switched out the underlying implementation before. There's no reason for consumers to need access to this. Add the `_` back.